examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
```

Comment:
We already have plenty of examples demonstrating the polygon classes; no need for a new one.